app/server/utils/user.py

```diff
@@ -850,7 +850,7 @@ def create_user_without_transfer_account(phone):
                 phone=phone,
                 registration_method=RegistrationMethodEnum.USSD_SIGNUP)
 
-    organisation = g.active_organisation
+    organisation = Organisation.query.get(2)
```

Comment:
Why is this necessary?

Reply:
I keep forgetting to clean this out, I find myself having to do this because of the fact that I cannot create a user using the external_auth process because its actually broken in the local environment. It really pisses me off.
I shall revert this though

Reply:
The organisation stuff is a fucking pain in the ass.

Reply:
Yeah. It really is irksome